### PR TITLE
[DC-3445] Add `raw_fitbit_qc.py` for `steps_intaday`

### PR DESCRIPTION
**Pull request by hiro-mishima**

### CODE REVIEW

File: data_steward/analytics/cdr_ops/raw_fitbit_qc.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+impersonation_creds = auth.get_impersonation_credentials(
+    run_as, target_scopes=IMPERSONATION_SCOPES)
+
+client = BigQueryClient(project_id, credentials=impersonation_creds)
+
+# # STEPS_INTRADAY table
+
+# Validation criteria for steps_intraday is the following:
+# - Field names and data types match the approved schema's
+# - The table includes both PTSC and CE data per the src_id field
```
Comment (lledezma) [inline data_steward/analytics/cdr_ops/raw_fitbit_qc.py:44]: I don't think the first check applies anymore.

**Review by lledezma — COMMENTED** [review]
The query works and I like the check status message, but check does not display any results or status.

**Comment by hiro-mishima** [comment]
Thanks for reviewing, @lledezma! I just pushed the updated version. Let me know if it suffices.

**Review by lledezma — APPROVED** [review]
Thanks for the updates Hiro! You could provide an alias for the src_id count if you think I would help the viewer, but you don't have to.

**Pull request merged**